dev/src/index.ts

```diff
@@ -76,6 +76,7 @@ const serviceConfig = interfaces['google.firestore.v1.Firestore'];
 
 import api = google.firestore.v1;
 import {CollectionGroup} from './collection-group';
+import {DocumentData} from '@google-cloud/firestore';
```

Comment:
This is already imported. If you don't like typing `firestore.DocumentData` you should instead do `const DocumentData = firestore.DocumentData`.